changelogs/unreleased/2870-resource-id-in-table.yml

```diff
@@ -0,0 +1,4 @@
+description: Show resource id without version in resources table
+issue-nr: 2870
+change-type: patch
+destination-branches: [master, iso5]
```

Comment:
Can't this easily go into iso4? iirc, this sprint was mostly about fixing small stuff that could go into iso4 as well.